lib/cog/commands/cat.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Cog.Commands.Cat do
+  use Cog.Command.GenCommand.Base,
+    bundle: Cog.Util.Misc.embedded_bundle
+
+  alias Cog.Command.Service.DataStore
+
+  # Note, we use the `tee` namespace because the namespace we read from
+  # must be the same one that data was written into.
+  @data_namespace [ "commands", "tee" ]
+
+  @description "Save and pass through pipeline data"
+
+  @long_description """
+  The cat command retrieves pipeline output that was previously saved using the tee command.
+  """
+
+  @arguments "<name>"
+
+  rule "when command is #{Cog.Util.Misc.embedded_bundle}:cat allow"
+
+  option "merge", short: "m", type: "bool", required: false,
+         description: "Merge current pipeline map into saved pipeline map"
+
+  option "append", short: "a", type: "bool", required: false,
+         description: "Append current pipeline output to saved pipeline data, returning an array"
+
+  option "insert", short: "i", type: "string", required: false,
+         description: "Insert current pipeline output into saved pipeline map as the field specified for this option"
+
+  def handle_message(%{options: %{"merge" => true, "append" => true}} = req, state) do
+    {:error, req.reply_to, "The append and merge options cannot be specified together", state}
+  end
+
+  def handle_message(%{args: [key], options: opts} = req, state) do
+    case DataStore.fetch(@data_namespace, key) do
+      {:ok, data} ->
+        cond do
+          opts["insert"] ->
+            handle_transform(:insert, req, data, state)
+          opts["append"] ->
+            {:reply, req.reply_to, List.wrap(data) ++ List.wrap(req.cog_env), state}
```

Comment:
It's appending the current pipeline output to the saved pipeline output. 

Example:

```
@ash seed '{ "foo": "value1" }' | tee foo
```
```
[
  {
    "foo": "value1"
  }
]
```
```
@ash seed '{ "foo": "value2" }' | cat -a foo
```
```
[
  {
    "foo": "value1"
  },
  {
    "foo": "value2"
  }
]
```